server/app/services/webhook_service.py

```
@@ -854,3 +858,345 @@ async def _send_webhook_to_client(
+        """
+        Find service record for a potential callback from missed outbound call.
+        
+        Args:
+            phone_number: Customer phone number
+            org_id: Organization ID
+            db: Database session
+            
+        Returns:
+            Service record ID if found, None otherwise
+        """
+        from app.models import ServiceRecord, Call
```

Ruff (0.14.7)

Redefinition of unused `ServiceRecord` from line 14

Remove definition: `ServiceRecord`

(F811)